openssl/src/symm.rs

```diff
@@ -307,6 +307,31 @@ impl Cipher {
         unsafe { Cipher(ffi::EVP_seed_ofb()) }
     }
 
+    #[cfg(all(any(ossl111, libressl291), not(osslconf = "OPENSSL_NO_SM4")))]
```

Comment:
Can you add these here as well? https://github.com/sfackler/rust-openssl/blob/master/openssl/src/cipher.rs#L102

Reply:
Sure. Thanks for your comments :-)